lib/runtime/procs/source.js

```diff
@@ -16,12 +15,19 @@ var INFO = {
     options: {
         from: {},
         to: {},
-        last: {}
+        last: {},
+        queueSize: {}
```

Comment:
This is the expected format for the (still unused) autocompleter.
Until we get that teed up again I'd just consider this a no-op.